### PR TITLE
Add new parameters for Processing algorithms

### LINKED ISSUE #7713: Add relations export option to package alg (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#49520
Author: @elpaso
QGIS version: 3.28

**Add relations export option to package alg**

### PR Description:
Adds an option to the package algorithm to follow project relations and export related tables following the hierarchy upstream and downstream, taking selected features into account.

immagine

Funded by: ARPA Piemonte

### Commits tagged with [need-docs] or [FEATURE]

### LINKED ISSUE #7778: Use GEOS >= 3.9.0 fixed precision (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#50148
Author: @nyalldawson
QGIS version: 3.28

**Use GEOS >= 3.9.0 fixed precision**

### PR Description:
Addresses review comments from https://github.com/qgis/QGIS/pull/46570 , so that this PR can be merged...

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by DelazJ**
Package layers (fixes #7713)
Add Grid size parameter to Overlay algorithms (fixes #7778)

### CODE REVIEW

File: docs/user_manual/processing_algs/qgis/vectoranalysis.rst [hunk 1]
```diff
@@ -1360,6 +1363,28 @@ Parameters
...
+   * - **Grid size**
+
+       Optional
+     - ``GRID_SIZE``
+     - [number]
+
+       Default: *Unset*
+     - If provided, the input geometries are snapped to a grid of the given size,
+       and the result vertices are computed on that same grid. Requires GEOS 3.9.0 or higher.
+
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1387]: @lbartoletti If ever you have more/better explanation to put here...

Comment (lbartoletti) [reply docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1387]: For me it's ok, simple and an user-friendly description.

**Review by lbartoletti — APPROVED** [review]

File: docs/user_manual/processing_algs/qgis/vectoranalysis.rst [hunk 2]
```diff
@@ -1360,6 +1363,28 @@ Parameters
...
+     - Name
+     - Type
+     - Description
+   * - **Grid size**
+
+       Optional
+     - ``GRID_SIZE``
+     - [number]
+
+       Default: *Unset*
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1384]:
```suggestion
       Default: Not set
```

File: docs/user_manual/processing_algs/qgis/vectoroverlay.rst [hunk 3]
```diff
@@ -994,6 +1054,15 @@ Advanced parameters
      - [string]
      - Add a prefix to identify fields of the overlay layer.
        Duplicate field names will be appended a count suffix to avoid collision.
+   * - **Grid size**
+
+       Optional
+     - ``GRID_SIZE``
+     - [number]
+
+       Default: *Unset*
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectoroverlay.rst:1063]:
```suggestion
       Default: Not set
```

File: docs/user_manual/processing_algs/qgis/vectoroverlay.rst [hunk 4]
```diff
@@ -868,6 +919,15 @@ Advanced parameters
...
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectoroverlay.rst:928]:
```suggestion
       Default: Not set
```

File: docs/user_manual/processing_algs/qgis/vectoroverlay.rst [hunk 5]
```diff
@@ -467,6 +489,35 @@ Parameters
...
+     - [string]
+     - Add a prefix to identify fields of the overlay layer.
+       Duplicate field names will be appended a count suffix to avoid collision.
+   * - **Grid size**
+
+       Optional
+     - ``GRID_SIZE``
+     - [number]
+
+       Default: *Unset*
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectoroverlay.rst:517]:
```suggestion
       Default: Not set
```

File: docs/user_manual/processing_algs/qgis/vectoroverlay.rst [hunk 6]
```diff
@@ -166,6 +170,28 @@ Parameters
...
+     - Name
+     - Type
+     - Description
+   * - **Grid size**
+
+       Optional
+     - ``GRID_SIZE``
+     - [number]
+
+       Default: *Unset*
```
Comment (DelazJ) [inline docs/user_manual/processing_algs/qgis/vectoroverlay.rst:191]:
```suggestion
       Default: Not set
```

File: docs/user_manual/processing_algs/qgis/vectoranalysis.rst (hunk 1 above)
Comment (DelazJ) [reply docs/user_manual/processing_algs/qgis/vectoranalysis.rst:1387]: and picked in PostGIS docs 😄

**Pull request merged**